src/test/script_standard_tests.cpp

```diff
@@ -365,7 +365,7 @@ BOOST_AUTO_TEST_CASE(script_standard_IsMine)
         // Keystore has key
         keystore.AddKey(keys[0]);
         result = IsMine(keystore, scriptPubKey);
-        BOOST_CHECK_EQUAL(result, ISMINE_SPENDABLE);
+        BOOST_CHECK((result & ISMINE_SPENDABLE) != ISMINE_NO);
```

Comment:
```suggestion
        BOOST_CHECK_EQUAL(result, ISMINE_LEGACY_SPENDABLE);
```
and similarly for all the other checks in this test.